contract/r/gnoswap/v1/router/exact_in.gno

```diff
@@ -84,16 +85,20 @@ func ExactInSwapRoute(cur realm,
 		panic(err)
 	}
 
-	if params.IsUnwrap() {
-		amount, err := strconv.ParseInt(params.exactAmount, 10, 64)
-		if err != nil {
-			panic(err)
-		}
+	amount, err := strconv.ParseInt(outputAmount, 10, 64)
+	if err != nil {
+		panic(err)
+	}
 
+	amount = amount * -1
```

Comment:
This value will be handled in `unwrapWithTransfer` function and `finalizeSwap` functions. But should be double-check